services/cd-service/pkg/httperrors/httperrors.go

```diff
@@ -35,3 +35,9 @@ func PublicError(ctx context.Context, err error) error {
 	logger.Error("grpc.public", zap.Error(err))
 	return status.Error(codes.InvalidArgument, "error: "+err.Error())
 }
+
+func AuthError(ctx context.Context, err error) error {
+	logger := logger.FromContext(ctx)
+	logger.Error("grpc.auth", zap.Error(err))
```

Comment:
This is hard against our engineering standards: handle OR log an error but not both.

Reply:
done